zorder/src/main/scala/org/locationtech/sfcurve/zorder/Z3.scala

```diff
@@ -143,13 +143,13 @@ object Z3 {
           zranges(min, nextOffset, 7, nextLevel)
         } else {
           // bottom out - add the entire range so we don't miss anything
-          mq += IndexRange(octRange.min.z, octRange.max.z, false)
+          mq += IndexRange(octRange.min.z, octRange.max.z, contained = false)
         }
       }
     }
 
     // kick off recursion over the narrowed space
-    zranges(commonPrefix, TOTAL_BITS - commonBits, 0, 0)
+    zranges(commonPrefix, 64 - commonBits, 0, 0)
```

Comment:
@elahrvivaz is there a reason to switch from a global constant to the hardcoded 64 here and below?

I'm not pushing too hard here; just curious.